code/game/objects/items/grenades/grenade.dm

```diff
@@ -15,7 +16,8 @@
 	max_integrity = 40
 	var/active = 0
 	var/det_time = 50
-	var/display_timer = 1
+	var/display_timer = TRUE
+	var/want_timer = TRUE

```

Comment:
`display_timer` is already responsible for whether to print the detonation message on examine, unless I'm missing something here. `want_timer` is redundant.

Reply:
It does if you only look at the grenade code. But the chemical grenade sets that value during runtime, thus i added another var